e2e/custom_module/blazeface/yarn.lock

```diff
@@ -117,47 +117,51 @@
   resolved "https://registry.yarnpkg.com/@tensorflow-models/blazeface/-/blazeface-0.0.5.tgz#6261fcc5de61a283ea409b69f6a851f44a809826"
   integrity "sha1-YmH8xd5hooPqQJtp9qhR9EqAmCY= sha512-bIMJDV2CD8rr7v9OrIDdDZVh9sbk0EhlZdHaDFpgWUw3Lp21wUN7GSifOJVE5hV2ngmpzM+J12OH8bG1GZoDag=="
 
+"@tensorflow/tfjs-backend-cpu@link:../../../link-package/node_modules/@tensorflow/link-package/node_modules/@tensorflow/tfjs-backend-cpu":
```

Comment:
Is it intended to update yarn.lock files?

Reply:
yarn.lock files are automatically updated after running `yarn test-ci`